### PR TITLE
FIX and NEW i18n for creating tickets via the mail collector

### PR DESCRIPTION
**Pull request by ymollard**
# FIX
- Missing require since for https://github.com/Dolibarr/dolibarr/pull/19601

# NEW
- i18n of default mail collectors (examples)
- Set MAIN_EMAILCOLLECTOR_MAIL_WITHOUT_HEADER default to 1 (Benefit: more readable conversations in the ticket history)
- Add a switch to disable MAIN_EMAILCOLLECTOR_MAIL_WITHOUT_HEADER in the admin UI of the mail collector
- Rephrased some en_US labels to be clearer
- Use default  values for sender e-mail, message introduction and message signature so that answers to tickets can be sent with no prior config in admin settings

### CODE REVIEW

File: htdocs/admin/emailcollector_list.php [hunk 1]
```diff
@@ -594,6 +596,33 @@
...
+print "</tr>\n";
+
+// Hide e-mail headers from collected messages
+print '<tr class="oddeven"><td>'.$langs->trans("EmailCollectorHideMailHeaders").'</td>';
+print '<td class="left">';
+if ($conf->use_javascript_ajax) {
+	print ajax_constantonoff('MAIN_EMAILCOLLECTOR_MAIL_WITHOUT_HEADER');
+} else {
+	$arrval = array('0' => $langs->trans("No"), '1' => $langs->trans("Yes"));
+	print $formcategory->selectarray("MAIN_EMAILCOLLECTOR_MAIL_WITHOUT_HEADER", $arrval, $conf->global->TICKET_AUTO_READ_WHEN_CREATED_FROM_BACKEND);
```
Comment (eldy) [inline htdocs/admin/emailcollector_list.php:617]: You should use 
$form->selectarray(

File: htdocs/admin/emailcollector_list.php [hunk 2]
```diff
@@ -594,6 +596,33 @@
 print '</table>'."\n";
 print '</div>'."\n";
 
+$formcategory = new FormCategory($db);
```
Comment (eldy) [inline htdocs/admin/emailcollector_list.php:599]: No need of $formcategory (a specialiaed class to manage categories). 
You can use $form that is always defined

File: htdocs/admin/emailcollector_list.php [hunk 3]
```diff
@@ -29,6 +29,8 @@
 require_once DOL_DOCUMENT_ROOT.'/core/class/html.formcompany.class.php';
 require_once DOL_DOCUMENT_ROOT.'/core/lib/date.lib.php';
 require_once DOL_DOCUMENT_ROOT.'/core/lib/company.lib.php';
+require_once DOL_DOCUMENT_ROOT."/core/class/html.formcategory.class.php";
```
Comment (eldy) [inline htdocs/admin/emailcollector_list.php:32]: No need of $formcategory (a specialiaed class to manage categories). 
You can use $form that is always defined

File: htdocs/admin/ticket.php [hunk 4]
```diff
@@ -644,7 +648,7 @@
 print '</tr>';
 
 // Email for notification of TICKET_CREATE
-print '<tr class="oddeven"><td>'.$langs->trans("TicketEmailNotificationTo").' ('.$langs->trans("Creation").')</td>';
```
Comment (eldy) [inline htdocs/admin/ticket.php:647]: Why removing the 
`' ('.$langs->trans("Creation").')`
It here to be clear that this email is used on ticket creation vs the next point
`.' ('.$langs->trans("Responses").')'`
used on email responses.

Comment (ymollard) [reply htdocs/admin/ticket.php:647]: Just `(creation)` in brackets looks unclear to me.
I'll make another proposal to include "creation" in a complete sentence into `TicketEmailNotificationTo`

File: htdocs/langs/fr_FR/ticket.lang [hunk 5]
```diff
@@ -90,10 +90,10 @@ TicketPublicAccess=Une interface publique ne nécessitant aucune identification
 TicketSetupDictionaries=Les types de ticket, sévérité et codes analytiques sont paramétrables à partir des dictionnaires
 TicketParamModule=Configuration des variables du module
 TicketParamMail=Configuration de la messagerie
-TicketEmailNotificationFrom=Email from de notification
-TicketEmailNotificationFromHelp=Utilisé dans les messages de réponses des tickets par exemple
-TicketEmailNotificationTo=E-mail de notification à
-TicketEmailNotificationToHelp=Envoyer des notifications par e-mail à cette adresse.
+TicketEmailNotificationFrom=Adresse e-mail émettrice des réponses aux tickets
```
[the remark this answers is not in the document]
Comment (ymollard) [reply htdocs/langs/fr_FR/ticket.lang:93]: Yes. Done to make testing handy and keep record of the translation I proposed for a further update of Transiflex :)

File: htdocs/admin/emailcollector_list.php (hunk 1 above)
Comment (ymollard) [reply htdocs/admin/emailcollector_list.php]: Fixed in https://github.com/Dolibarr/dolibarr/pull/20669/commits/bdfe7f12d90051e004dc2c9a715d0a7aeb0868c7

File: htdocs/admin/emailcollector_list.php (hunk 3 above)
Comment (ymollard) [reply htdocs/admin/emailcollector_list.php]: Also fixed in https://github.com/Dolibarr/dolibarr/pull/20669/commits/bdfe7f12d90051e004dc2c9a715d0a7aeb0868c7

File: htdocs/admin/emailcollector_list.php (hunk 2 above)
Comment (ymollard) [reply htdocs/admin/emailcollector_list.php]: Also fixed in https://github.com/Dolibarr/dolibarr/pull/20669/commits/bdfe7f12d90051e004dc2c9a715d0a7aeb0868c7

**Pull request merged**